evil-collection-debbugs.el

```diff
@@ -49,7 +49,7 @@
     ;; filter
     (kbd "s") 'debbugs-gnu-narrow-to-status
     ;; "S" 'debbugs-gnu-widen ; Useless if we can just press "s RET" (empty filter).
-    "S" 'debbugs-gnu-toggle-suppress
+    "x" 'debbugs-gnu-toggle-suppress
```

Comment:
Why not.
In general many toggle keys are z-prefixed.  What do you think about `zs` or `zh`?

Reply:
I think x is open and it was the original keybinding. I don't have an opinion otherwise though so we can put it at zs if you think that's better.

Reply:
Fine with `x`, bindings are quite simple so far.  We could always move it to `zs` if bindings get more crowded.